canvas_modules/common-canvas/package.json

```diff
@@ -105,8 +105,8 @@
     "carbon-components": "^10.50.0",
     "carbon-components-react": "^7.50.0",
     "carbon-icons": "^7.0.7",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
+    "react": "^18.0.0",
```

Comment:
I think we also want to support 16 yet.  Can we have 16 and 18 as supported versions?